src/feature/flow_chart_panel/manifest/PARAMETERS_MANIFEST.ts

```diff
@@ -1,12 +1,20 @@
 import manifests from "@src/data/manifests-latest.json";
-type FunctionParametersType = {
-  [key: string]: {
-    [key: string]: {
-      type: string;
-      default: string | number;
-      options?: string[];
-    };
-  };
-};
+import { z } from "zod";
```

Comment:
I've never used zod before, @itsjoeoui can probably explain

Reply:
@smahmed776 As great as all the TypeScript types definition help us during development, TypeScript only checks type at compile time and there is no runtime type checking. `zod` fixes this problem by bringing runtime type checking to TypeScript. Since we are loading the manifests.json at runtime, this adds another layer of satefy on top!

Reply:
It's interesting to know that, although I have doubts about whether we actually need that runtime check since it is auto-generated. Instead, we can focus on enhancing the Python function responsible for generating this file. Nonetheless, since it is a small package, it may not be a major concern.